### PR TITLE
OMA-828 - Hardcode ad choices content if nothing comes from APIv3 and for custom markup

### PR DESCRIPTION
**Pull request by PabloGarciaAldehuela**

### CODE REVIEW

File: hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/Ad.java [hunk 1]
```diff
@@ -51,6 +51,10 @@
     private static final String PN_IMPRESSION_QUERY_PARAM = "t";
     private static final int MIN_POINTS = 10;
 
+    private static final String HC_CONTENT_INFO_LINK_URL = "https://pubnative.net/content-info";
```
Comment (eros902002) [inline hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/Ad.java:54]: What's HC? : )

File: hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/Ad.java [hunk 2]
```diff
@@ -210,6 +214,7 @@ public View getContentInfo(Context context) {
         AdData data = getMeta(APIMeta.CONTENT_INFO);
         if (data == null) {
             Log.e(TAG, "getContentInfo - contentInfo data not found");
+            return hardcodeContentInfo(context, result);
```
Comment (eros902002) [inline hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/Ad.java:217]: I would not call it hardcoded. We can just say: getDefaultContentInfo

**Review by eros902002 — APPROVED** [review]
Looks good : )

File: hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/Ad.java (hunk 2 above)
Comment (PabloGarciaAldehuela) [reply hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/Ad.java]: Just changed it, thanks!

File: hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/Ad.java (hunk 1 above)
Comment (PabloGarciaAldehuela) [reply hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/Ad.java]: Changed it, as it doesnt look good to have something mentioning hardcoded (HC) there

**Pull request merged**